java/apirag-server/src/main/resources/functions/plotfunction1d.json

```diff
@@ -31,8 +31,12 @@
       "title": {
         "description": "The title for the chart.",
         "type": "string"
+      },
+      "caption": {
+        "description": "A caption that describes this data visualization and how it answers the user's question.",
+        "type": "string"
```

Comment:
What is the purpose for this caption? We currently don't display an Assistant text content next to the graph to the user.

Reply:
I added the caption to the javascript code as well, so it is displayed.
The reason I added this: I've had a number of chat sessions where I asked questions like "What credit card would you recommend for me" and the answer would be a bar chart comparing the rewards I would earn on the different cards, but without any explanation. While the visual is useful, it needs some text to explain what the user is seeing. That's what I added here and it seems to work well in my (limited) testing.

Reply:
ok, but this only works for the openai API compatible models for now, since for Bedrock Llama3, I only accept function calls which are pure JSON and no other content.